modules/integration/tests-common/common-artifacts/lazy-loading-info/src/main/java/org/wso2/appserver/integration/common/artifacts/lazyloading/LazyLoadingInfoUtil.java

```diff
@@ -0,0 +1,164 @@
+    private static final Log log = LogFactory.getLog(LazyLoadingInfoUtil.class);
+    private static final String CARBON_WEBAPPS_HOLDER_LIST = "carbon.webapps.holderlist";
+    private static final String WEBAPPS = "webapps";
+    private static final String GHOST_WEB_APP = "GhostWebApp";
+
+
+    /**
+     * Get the server configuration context.
+     *
+     * @return configuration context of the server.
+     */
+    private static ConfigurationContext getServerConfigurationContext() {
+        ConfigurationContextService configurationContext =
+                (ConfigurationContextService) PrivilegedCarbonContext.getThreadLocalCarbonContext().
+                        getOSGiService(ConfigurationContextService.class, null);
+        return configurationContext.getServerConfigContext();
+    }
+
+
+    /**
+     * Get the configuration contexts of all loaded tenants
+     *
+     * @return Map that contains the  configuration contexts
+     */
+    private static Map<String, ConfigurationContext> getTenantConfigServerContexts() {
+        return TenantAxisUtils.getTenantConfigurationContexts(getServerConfigurationContext());
+    }
+
+
+    /**
+     * Get the configuration context of given tenant.
+     *
+     * @param tenantDomain tenant domain name.
+     * @return ConfigurationContext of given tenant
+     */
+    private static ConfigurationContext getTenantConfigurationServerContext(String tenantDomain) {
+        return getTenantConfigServerContexts().get(tenantDomain);
+    }
+
+    /**
+     * Check  the given tenant is loaded.
+     *
+     * @param tenantDomain Domain name of the tenant
+     * @return TenantStatus with current status  information about the tenant.
+     */
+    protected static TenantStatus getTenantStatus(String tenantDomain) {
+        boolean isTenantContextLoaded = false;
+        Map<String, ConfigurationContext> tenantConfigServerContexts = getTenantConfigServerContexts();
+        if (tenantConfigServerContexts != null) {
+            isTenantContextLoaded = tenantConfigServerContexts.containsKey(tenantDomain);
+            log.info("Tenant " + tenantDomain + " loaded :" + isTenantContextLoaded);
+        }
+        return new TenantStatus(isTenantContextLoaded);
+    }
+
+    /**
+     * Check  the given web-app of given tenant is loaded.
+     *
+     * @param tenantDomain Tenant name
+     * @param webAppName   Web-app Name
+     * @return WebAppStatus  with current status  information about the Web app.
+     */
+    protected static WebAppStatus getWebAppStatus(String tenantDomain, String webAppName) {
+        WebAppStatus webAppStatus = new WebAppStatus();
+        ConfigurationContext tenantConfigurationServerContext = getTenantConfigurationServerContext(tenantDomain);
+        if (tenantConfigurationServerContext != null) {
+            webAppStatus.setTenantStatus(new TenantStatus(true));
+            log.info("Tenant " + tenantDomain + " configuration context is loaded.");
```

Comment:
@prabathabey No, these should be info log because  this REST service is used by the test classes to verify the status of  the web app and tenant. So in a test failed scenario having these kind of info logs helps to identify the actual  cause for the failure.